frontend/src/features/moti/mapSideBar/components/InventoryFormButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from 'components/common/form';
+import * as React from 'react';
+import styled from 'styled-components';
+
+const ButtonWrapper = styled.div`
```

Comment:
Don't change, but I find it makes the most sense to have the actual component at the top of the file rather than dependent sub-component.s